src/timetracker/components/create-project-page/CreateProjectPage.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import "./CreateProjectPage.css";
+import CreateProjectDialog from "./CreateProjectDialog";
+
+const CreateProjectPage = () => {
+  let projects = [
+    {id:1,name:"project1",hours:'2.00h'},
+    {id:2,name:"project2",hours:'55.56h'},
+    {id:3,name:"project3",hours:'23.00h'}
+  ]
+ let [open,setOpen] = useState(false)
+ let clickHandler=()=>{
+  (open)?setOpen(false):setOpen(true)
+ }
+  return (
+    
+    <div className="create-project h-screen flex flex-col items-center">
+      {(open)?
+    <CreateProjectDialog open={open} setOpen={setOpen}/>:''}
+      <div className="mt-36">
+        <h1 className="font-bold text-light text-[48px]">Projects</h1>
+      </div>
+      <div className="Button">
+        <button className="bg-secondary text-light py-2 px-4 rounded-md font-semibold font-[16px] m-8 shadow-md " onClick={clickHandler} >
+          Create New Project
+        </button>
+    
+      </div>
```

Comment:
The button on the wireframe is aligned to flex-start on the desktop.

Maybe add a max width to the overall container for the button and the table. Then do something like `md:self-start` to make it work. Or just align everything to start if it's just the button and table, since table is going to be full width anyway.